scripts/base_network.py

```diff
@@ -121,7 +121,9 @@ def _load_buses_from_osm():
 
     buses = buses.loc[:, ~buses.columns.str.contains("^Unnamed")]
     buses["v_nom"] /= 1e3
-    buses["carrier"] = buses.pop("dc").map({True: "DC", False: "AC"})
+    # All carriers are temporary set to "AC" to avoid the mixed-carries problem
+    # buses["carrier"] = buses.pop("dc").map({True: "DC", False: "AC"})
+    buses["carrier"] = "AC"
```

Comment:
Add a TODO: remove to enable HVDC buses and lines, once probably debugged. It is good to have TODOs when known problems are known

Reply:
Yes, that's absolutely makes sense. Done.